### PR TITLE
[NU-1778] Add comment action in ScenarioActivity API

### PR DESCRIPTION
**Pull request by mgoworko**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala [hunk 1]
```diff
@@ -58,6 +57,11 @@ object Dtos {
           displayableName = "Delete",
           icon = "/assets/activities/actions/delete.svg"
         ),
+        ScenarioActivityActionMetadata(
+          id = "add_comment",
+          displayableName = "Add",
```
Comment (mgoworko) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala:62]: Will it be used by FE? Is it ok?

Comment (Dzuming) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala:62]: I think the `Add comment` fits better here. We will display this text on icon hover as a tooltip

Comment (mgoworko) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala:62]: What about other comment-related actions? Should they have " comment" suffix too?

Comment (Dzuming) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala:62]: Yes, let's also add this suffix in other cases

Comment (mgoworko) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala]: Ok added. + I imporve date formatting and additional fields for batch

**Pull request merged**